android/java/org/chromium/chrome/browser/playlist/sources.gni

```diff
@@ -5,6 +5,96 @@
 
 brave_app_playlist_deps = [
   "//brave/third_party/android_deps:androidx_media3_media3_exoplayer_hls_java",
-  "//brave/third_party/android_deps:com_brave_playlist_java",
+  "//brave/third_party/android_deps:androidx_room_room_runtime_java",
+  "//third_party/android_deps:material_design_java",
+  "//third_party/android_deps:org_jetbrains_kotlinx_kotlinx_coroutines_core_jvm_java",
+  "//third_party/androidx:androidx_constraintlayout_constraintlayout_java",
   "//third_party/androidx:androidx_media3_media3_common_java",
+  "//third_party/androidx:androidx_media3_media3_datasource_java",
+  "//third_party/androidx:androidx_media3_media3_exoplayer_java",
+  "//third_party/androidx:androidx_media3_media3_session_java",
+  "//third_party/androidx:androidx_media3_media3_ui_java",
+  "//third_party/androidx:androidx_transition_transition_java",
 ]
+
+brave_playlist_sources = []
+
+playlist_kotlin_sources = [
```

Comment:
please put these in their own BUILD.gn as an `android_library` target and add to deps instead of sources

Reply:
Fixed in https://github.com/brave/brave-core/pull/26628/commits/e257593acc6cbfaa8c06228c44d674fc0dc3b97b